lib/ClangImporter/ClangImporter.cpp

```diff
@@ -4127,7 +4122,7 @@ void ClangImporter::Implementation::lookupVisibleDecls(
     DeclBaseName name = baseName.toDeclBaseName(SwiftContext);
     if (!lookupValue(table, name, consumer) &&
         SwiftContext.LangOpts.EnableExperimentalEagerClangModuleDiagnostics) {
-      diagnoseValue(table, name);
```

Comment:
This was actually a bug in that this lookup could return both a member and top level declaration, when only the top level declaration is desired.